README.md

```diff
@@ -4,123 +4,32 @@
 
 # Otomi Core
 
-Otomi Core is the Heart of the Otomi Container Platform. Otomi Container Platform offers an out-of-the-box enterprise container management platform (on top of Kubernetes) to increase developer efficiency and reduce complexity. It is a turnkey cloud native solution that integrates upstream Kubernetes with proven open source components. Otomi is made available as a single deployable package with curated industry proven applications and policies for better governance and security. With carefully crafted sane defaults at every step, it minimizes configuration effort and time to market. Otomi automates most (if not all) of your cluster operations and includes application lifecycle management at its core. It is open source and transparent, allowing customization but also extensibility. Incorporating Open Source standards and best practices, Otomi aims to bring new features and stability with every iteration.
+Otomi Core is the heart of the Otomi Container Platform, which extends Kubernetes with an advanced ingress architecture, curated industry proven applications and policies for better governance and security.
 
-Important features:
+Otomi aims to support the most common DevOps use cases and strongly relies on GitOps pattern, where desired state is reflected as code and any change to that manifest is reflected at the platform.
```

Comment:
I don't think that is really what people perceive as gitops, and maybe we should rename it to ConfOps. GitOps is described as code being synced from git, but we have all code in containers and only conf is left in git.

Anyways, this line does not belong here in this list so please remove it here. Maybe you want to add a small section below on ConfOps

Reply:
```suggestion
```

Reply:
Maybe I can describe it as configuration as code that follows GitOps pattern